pandas/core/nanops.py

```diff
@@ -1332,31 +1332,34 @@ def nancorr(
 
 
 def get_corr_func(method):
-    if method in ["kendall", "spearman"]:
-        from scipy.stats import kendalltau, spearmanr
-    elif method in ["pearson"]:
-        pass
+    if method == "kendall":
+        from scipy.stats import kendalltau
+
+        def func(a, b):
+            return kendalltau(a, b)[0]
+
+        return func
+    elif method == "spearman":
+        from scipy.stats import spearmanr
+
+        def func(a, b):
+            return spearmanr(a, b)[0]
+
+        return func
+    elif method == "pearson":
+
+        def func(a, b):
+            return np.corrcoef(a, b)[0, 1]
+
+        return func
     elif callable(method):
         return method
     else:
         raise ValueError(
-            f"Unknown method '{method}', expected one of 'kendall', 'spearman'"
+            f"Unknown method '{method}', expected one of "
+            "'kendall', 'spearman', 'pearson', or callable"
         )
 
-    def _pearson(a, b):
-        return np.corrcoef(a, b)[0, 1]
-
-    def _kendall(a, b):
-        # kendallttau returns a tuple of the tau statistic and pvalue
```

Comment:
is this comment worth retaining?

Reply:
Might as well, should we also have a comment for spearmanr and np.corrcoef?